journal-of-the-american-adacemy-of-audiology.csl

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Journal of the American Academy of Audiology</title>
+    <title-short>JAAA</title-short>
+    <id>http://www.zotero.org/styles/journal-of-the-american-adacemy-of-audiology</id>
+    <link href="http://www.zotero.org/styles/journal-of-the-american-adacemy-of-audiology" rel="self"/>
+    <link href="http://www.zotero.org/styles/genome-biology-and-evolution" rel="template"/>
+    <link href="http://www.audiology.org/publications-resources/journal-american-academy-audiology/instructions-authors" rel="documentation"/>
+    <author>
+      <name>O'Brien Patrick, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <category field="medicine"/>
+    <issn>1050-0545</issn>
+    <eissn>2157-3107</eissn>
+    <updated>2017-01-06T12:09:37+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor">
+      <name delimiter=", " name-as-sort-order="all" sort-separator=", " initialize-with="">
+        <name-part name="family" text-case="capitalize-first"/>
```

Comment:
I don't quite get what this is doing? names would typically be capitalize-first, no?